tui_main.c

```diff
@@ -1228,8 +1230,6 @@ void ged_adjust_bpm(Ged* a, Isz delta_bpm) {
   Isz new_bpm = (Isz)a->bpm + delta_bpm;
   if (new_bpm < 1)
     new_bpm = 1;
-  else if (new_bpm > 3000)
```

Comment:
This is an error. `orca --bpm 2147483647` and then increasing the BPM by 1 within the program will exceed `INT_MAX` and then invoke undefined behavior. In the best case, the value wraps to `INT_MIN` and the program freezes or crashes. In the worst case, memory is corrupted.

Reply:
oof yeah, sorry about that! Now it shouldn't be possible to overflow anymore -- limited bpm to INT_MAX